src/Categories/Categories.tsx

```diff
@@ -4,13 +4,14 @@ import './Categories.scss'
 
 interface CategoryProps {
   getCategory: (text: string) => void;
+  imageId: string
 }
 
-const Categories: React.FC<CategoryProps> = ({getCategory}) => {
+const Categories: React.FC<CategoryProps> = ({getCategory, imageId}) => {
```

Comment:
Probably could remove imageId since we figured out the right component to pass it down to 😂